audio-worklet/design-pattern/lib/wa2.js

```diff
@@ -113,16 +113,17 @@ class HeapAudioBuffer {
 
   /**
    * Returns a Float32Array object for a given channel index. If the channel
-   * index is undefined, it returns the reference to the base array of channel
+   * index is undefined, it returns the reference to the entire array of channel
    * data.
    *
    * @param  {number|undefined} channelIndex Channel index.
-   * @return {Float32Array|Array[Float32Array]} a channel data array or an
+   * @return {?Array} a channel data array or an
    * array of channel data.
    */
   getChannelData(channelIndex) {
-    if (channelIndex < this._channelCount)
+    if (channelIndex >= this._channelCount) {
```

Comment:
Haha. :-( Totally missed that in the first review.

Reply:
Heh. Let's fix it now.